panther_analysis_tool/schemas.py

```diff
@@ -20,7 +20,9 @@
 
 TYPE_SCHEMA = Schema(
     {
-        'AnalysisType': Or("datamodel", "global", "policy", "rule"),
+        'AnalysisType':
+            Or("datamodel", "global", "policy", "rule", "scheduled_rule",
+               "scheduled_query"),
```

Comment:
wondering why we include `scheduled_rule` here but not including its handling elsewhere / the schema for it?

Reply:
At the time `scheduled_rule`'s were not done being implemented with PAT, should be used in a few places now.